<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenCommande</class>
 <widget class="QWidget" name="FenCommande">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouvelle commande</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>208</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre de palettes de T1 :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="nNbT1">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nombre de palettes de T2 :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="nNbT2">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Num√©ro de quai :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cNumQuai">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Code client :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tCodeClient">
      <property name="maxLength">
       <number>9</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="bOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bCancel">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>FenCommande</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
